src/utils/hooks/useLoadApplicationNEW.ts

```diff
@@ -64,13 +86,24 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
       submissionMessage: application.template?.submissionMessage as string,
     }
 
+    const stages = data.applicationStageStatusAlls?.nodes as ApplicationStageStatusAll[]
```

Comment:
Are we using this anywhere in UI ? I was thinking maybe we would do a seperate query for this when accessing history ?

Reply:
We do need to get `stageID` and `stageNumber` to be used on the filter for applications list (that uses the same type `ApplicationDetails`) and when we display the Review page we need to know the stage title:
![Screen Shot 2021-02-23 at 10 49 30 AM](https://user-images.githubusercontent.com/16461988/108774739-e884f700-75c4-11eb-93ae-b7046dfa50cf.png)

 or the application Submission page the list of stages - which could be a second hook indeed:
<img width="721" alt="Screen Shot 2021-02-23 at 10 51 16 AM" src="https://user-images.githubusercontent.com/16461988/108774852-12d6b480-75c5-11eb-8f3d-0a8cf19a240d.png">

Reply:
Sure, looks like it's needed in summary, great.

Although current stage (for ApplicationPage and list), is available through application 

<img width="452" alt="Screen Shot 2021-02-23 at 11 04 09 AM" src="https://user-images.githubusercontent.com/26194949/108776172-e885f680-75c6-11eb-8f7f-aa4dcb4e605f.png">

Reply:
Oh, we need extra stuff. For application list we will also need the `statusHistoryTimeCreated` field which isn't an generated column on the application entity - that's when we decided to use the VIEW `application_stage_status_latest` so we didn't have to continue to map other stage/status attributes to it.